doc/guides/Dashboard.md

```diff
@@ -0,0 +1,216 @@
+# FOAM3 Dashboard Quick Start Guide
+
+This guide shows you how to create a simple dashboard in FOAM3 in just a few steps.
+
+## What You Need
+
+A FOAM3 dashboard requires these components:
+
+1. **Dashboard Class** - extends `foam.dashboard.view.DashboardView`
+2. **Widget Menus** - hidden menu entries that define your widgets
+3. **Citation Views** - for displaying individual records (optional)
+
+## Step 1: Create a Simple Dashboard
+
+Create `MyDashboard.js`:
+
+```javascript
+foam.CLASS({
+  package: 'my.dashboard',
+  name: 'MyDashboard',
+  extends: 'foam.dashboard.view.DashboardView',
+
+  properties: [
+    {
+      name: 'dashboardTitle',
+      value: 'My Dashboard'
+    },
+    {
+      name: 'main',
+      value: true
+    },
+    {
+      name: 'widgets',
+      factory: function() {
+        return {
+          'my.dashboard.users.table': { column: 6 },
+          'my.dashboard.users.count': { column: 6 }
+        };
+      }
+    }
+  ]
+});
+```
+
+## Step 2: Create Widget Menus
+
+Create `menus.jrl`:
+
+```javascript
+// Main dashboard menu
+p({
+  class: "foam.core.menu.Menu",
+  id: "my.dashboard",
+  label: "My Dashboard",
+  handler: {
+    class: "foam.core.menu.ViewMenu",
+    view: { class: "my.dashboard.MyDashboard" }
+  }
+})
+
+// Data table widget
+p({
+  class: "foam.core.menu.Menu",
+  id: "my.dashboard.users.table",
+  label: "Users Table",
+  handler: {
+    class: "foam.core.menu.ViewMenu",
+    view: {
+      class: "foam.dashboard.view.CardWrapper",
+      title: "Recent Users",
+      currentView: {
+        class: "foam.dashboard.view.DAOTable",
+        dao: "userDAO",
+        limit: 5,
+        viewMore: true,
+        viewMoreMenuItem: "users",
+        emptyTitle: "No Users",
+        emptySubTitle: "No users found"
+      }
+    }
+  },
+  parent: "hidden"
+})
+
+// Count widget
+p({
+  class: "foam.core.menu.Menu",
+  id: "my.dashboard.users.count",
+  label: "User Count",
+  handler: {
+    class: "foam.core.menu.ViewMenu",
+    view: {
+      class: "foam.dashboard.view.CardWrapper",
+      title: "Total Users",
+      currentView: {
+        class: "foam.dashboard.model.Count",
+        daoName: "userDAO"
+      }
+    }
+  },
+  parent: "hidden"
+})
+```
+
+## Step 3: Create Package File
+
+Create `pom.js`:
+
+```javascript
+foam.POM({
+  name: 'my.dashboard',
+  files: [
+    { name: "MyDashboard", flags: "js" }
+  ]
+});
+```
+
+## Step 4: Register Your Package
+
+Add to your main `pom.js`:
+
+```javascript
+foam.POM({
+  // ... other config
+  projects: [
+    // ... other projects
+    { name: "my.dashboard" }
+  ]
+});
+```
+
+## Step 5: Add Menu Entry
+
+Add to your main `menus.jrl`:
+
+```javascript
+p({
+  class: "foam.core.menu.Menu",
+  id: "my.dashboard",
+  label: "My Dashboard",
+  handler: {
+    class: "foam.core.menu.ViewMenu",
+    view: { class: "my.dashboard.MyDashboard" }
```

Comment:
Same as above, remove this, should be:
```
"widgets": {
        "my.dashboard.users.count": { column: 12 },
        "my.dashboard.users.table": { column: '1fr', SMColumn: 12, XSColumn: 12 }
      }

```

Reply:
we can allow mixed layout? 
SMColumn: 12, XSColumn: 12 will take full screen, and other than that use the fraction?

Reply:
In this case both would use 12 columns tho. Since the first one have 12, and the 2nd one will use the rest of the 2nd row, which is 12

Reply:
updated with extra info on that part, i hope i got it correctly